_data/projects/mautic.yml

```diff
@@ -0,0 +1,11 @@
+name: Mautic
+desc: An open source marketing automation platform.
+site: https://www.mautic.org
+tags:
+  - mautic
+  - marketing
+  - marketing-automation
+  - php
+upforgrabs:
+  name: good first issue
```

Comment:
Can you check if it is not having any extra space? Looks like this is throwing an error. @RCheesley

Reply:
Not that I can tell - where do you see there being an extra space?

It was passing until I changed the link to the one that you suggested - https://github.com/up-for-grabs/up-for-grabs.net/pull/3291/files/e320b414a88ea4dfecb7f608c62ea06533c10065#diff-e3cd3def82c43702c6c7065eb436326850a7b771107e54a22bc8919e96b56797 which I changed here: https://github.com/up-for-grabs/up-for-grabs.net/pull/3291/commits/50ab06e252236932c79c865b77b675d00c239c1e. Thanks!